src/components/DashboardWidgets/ClaimingWidget.tsx

```diff
@@ -80,44 +83,60 @@ const VotingPowerWidget = (): ReactElement => {
   const delegate = useDelegate()
   const { data: allocation } = useSafeTokenAllocation()
 
+  // TODO: Use real SEP #5 data
+  const sep5 = useMockSep5()
+  const isSep5Claimable = Number(sep5.claimable) > 0
+
   const totalClaimed = allocation?.vestingData.reduce((acc, { amountClaimed }) => {
     return acc.add(amountClaimed)
   }, BigNumber.from(0))
 
-  const hasUnredeemedAllocation = allocation?.vestingData.some(({ isExpired, isRedeemed }) => !isExpired && !isRedeemed)
+  const hasUnredeemedAllocation =
+    allocation?.vestingData.some(({ isExpired, isRedeemed }) => !isExpired && !isRedeemed) || isSep5Claimable
 
   const claimingSafeAppUrl = getGovernanceAppSafeAppUrl(safe.chainId.toString(), safe.safeAddress)
 
+  const hasClaimed = totalClaimed?.gt(0)
+
   return (
-    <>
-      <div>
-        <Title>Your voting power</Title>
-        <Button href={claimingSafeAppUrl} target="_blank" rel="noopener noreferrer" className={css.button}>
+    <Grid container p={1} height={1}>
+      <Grid item xs={12} display="flex" justifyContent="space-between" mb="auto">
+        <TypographyChip fontWeight={700} px={1}>
+          New allocation
```

Comment:
The Tooltip seems missing for the dashboard widget.

Reply:
Good catch! I've just added it.